tests/test_base.hpp

```diff
@@ -130,6 +130,16 @@ class server_config;
 
 class config_data
 {
+protected:
```

Comment:
Usually protected section writes at the bottom of class because it is implementation details.